pyproject.toml

```diff
@@ -24,7 +24,8 @@ classifiers = [
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
 dependencies = [
-    "strands-agents>=0.1.0,<1.0.0",
+    "strands-agents @ git+https://github.com/strands-agents/sdk-python.git",
```

Comment:
Currently dependency check fails due:

```
  Downloading a2a_sdk-0.2.11-py3-none-any.whl.metadata (4.8 kB)
ERROR: Cannot install a2a-sdk because these package versions have conflicting dependencies.

The conflict is caused by:
    a2a-sdk 0.2.12 depends on grpcio-tools>=1.60
    a2a-sdk 0.2.11 depends on grpcio-tools>=1.60
```

Dependencies will be updated after this change reach pypi: https://github.com/strands-agents/sdk-python/commit/1f64b4b819c412a684b677c17429604ecbcc66df

Reply:
Updated the dependencies to use SDK directly, expected to fail in GitHub CI.

Reply:
`test_swarm.py` is failing due to changes not published into pypi yet, will re-run CI after pypi release.